helpdesk_mgmt/models/helpdesk_ticket_team.py

```diff
@@ -18,7 +20,14 @@ class HelpdeskTeam(models.Model):
         default=lambda self: self.env['res.company']._company_default_get(
             'helpdesk.ticket')
     )
-
+    user_id = fields.Many2one('res.users', string='Team Leader',
```

Comment:
Why do you need to declare these fields if you are inheriting from `mail.alias.mixin`?

Reply:
I can separate this change in other pr... it has not relation with alias

Reply:
Yes, please. Each changeset per PR, so they can be discussed separately and one can be faster than the other.

Reply:
Done!